DEPENDENCIES.md

```diff
@@ -60,7 +60,8 @@ The following text applies to code linked from these dependencies:
 [uniffi](https://github.com/mozilla/uniffi-rs),
 [uniffi_bindgen](https://github.com/mozilla/uniffi-rs),
 [uniffi_build](https://github.com/mozilla/uniffi-rs),
-[uniffi_macros](https://github.com/mozilla/uniffi-rs)
+[uniffi_macros](https://github.com/mozilla/uniffi-rs),
+[zeitstempel](https://github.com/badboy/whatsys)
```

Comment:
This link seems to go to the wrong repo, should it be https://github.com/badboy/zeitstempel?

We get this value from [your Cargo.toml](https://github.com/badboy/zeitstempel/blob/280d75a82c66d7722fc840793ed54e2f61d14114/Cargo.toml#L8) which also looks like it might be pointing at the wrong repo. Since it's yours, I'm happy to ship like this and maybe fix it up later if needed, but wanted to double-check.

Reply:
Woops, yes, that's mine and I mixed up the repo URLs (because I did both crates at the same time). Let me fix that.